storage/storage-interface/src/lib.rs

```diff
@@ -423,7 +423,7 @@ impl MoveStorage for &dyn DbReader {
             })
             .collect::<Result<HashMap<_, AccountState>>>()?;
 
-        access_paths
+        access_path
```

Comment:
If we're not fetching a batch, we should probably simplify this function a bit since we don't need to deal with multiple access paths.

Reply:
@shikharvashistha, to elaborate up on what @phlip9 said, if we're only passing in a single access_path, this method can be heavily simplified (e.g., we no longer need to iterate or use collections). Moreover, the call sites can be simplified 😄

Reply:
Agreed ! 👍